napari/layers/image/_image_key_bindings.py

```diff
@@ -1,11 +1,70 @@
 from __future__ import annotations
 
+import napari
+
+from ...layers.utils.interactivity_utils import (
+    orient_plane_normal_around_cursor,
+)
 from ...layers.utils.layer_utils import register_layer_action
 from ...utils.translations import trans
 from ._image_constants import Mode
 from .image import Image
 
 
+def register_image_action(description: str):
+    return register_layer_action(Image, description=description)
+
+
+@Image.bind_key('z')
+@register_image_action(trans._('Orient plane normal along z-axis'))
+def orient_plane_normal_along_z(layer: Image):
+    orient_plane_normal_around_cursor(layer, plane_normal=(1, 0, 0))
+
+
+@Image.bind_key('y')
+@register_image_action(trans._('orient plane normal along y-axis'))
+def orient_plane_normal_along_y(layer: Image):
+    orient_plane_normal_around_cursor(layer, plane_normal=(0, 1, 0))
+
+
+@Image.bind_key('x')
+@register_image_action(trans._('orient plane normal along x-axis'))
+def orient_plane_normal_along_x(layer: Image):
+    orient_plane_normal_around_cursor(layer, plane_normal=(0, 0, 1))
+
+
+@register_image_action(trans._('orient plane normal along view direction'))
+def orient_plane_normal_along_view_direction(layer: Image):
+    viewer = napari.viewer.current_viewer()
+    if viewer.dims.ndisplay != 3:
+        return
+    layer.plane.normal = layer._world_to_data_ray(viewer.camera.view_direction)
+
+
+@Image.bind_key('o')
+def synchronise_plane_normal_with_view_direction(layer: Image):
```

Comment:
@alisterburt I absolutely love this mode, I'm wondering whether it should be a toggle control? But ditto to @kevinyamauchi, we can leave that as a feature request.